### PR TITLE
Packet Integration

### PR DESCRIPTION
**Pull request by gianarb**
**What this PR does / why we need it**:

It adds support for [Packet](https://metal.equinix.com)

**Which issue(s) this PR fixes**:

There is not issue.

**Special notes for your reviewer**:

I don't know what I am doing

**Release note**:

```improvement operator
improvement user Add support for Packet
```

### CODE REVIEW

File: dependency-versions.yaml [hunk 1]
```diff
@@ -42,6 +42,10 @@
           "repo": "https://github.com/gardener/gardener-extension-provider-openstack.git",
           "version": "v1.12.0"
         },
+        "provider-openstack": {
```
Comment (Diaphteiros) [inline dependency-versions.yaml:45]: copy-paste error: provider-packet

**Review by Diaphteiros — COMMENTED** [review]
I found one small copy-paste error, apart from that it looks good. I don't have packet credentials at hand to test it ...
Did you test it? You will have to choose a different DNS provider in your acre.yaml and deactivate the etcd backups for the virtual cluster etcd as well as for every seed (see here https://github.com/gardener/garden-setup/blob/master/docs/extended/iaas.md#seed-backup for the latter one). But all of that is configurable in your acre.yaml.

**Review by Diaphteiros — APPROVED** [review]
/lgtm

**Review by Diaphteiros — CHANGES_REQUESTED** [review]
Sorry, I somehow lost track of this PR. It looks good, I have just one minor request before I will merge it:
Please add the documentation for packet to the README.

**Comment by gianarb** [comment]
I am not sure I have never got this code actually running. And it looks like the garden setup will be replaced by something else. So I am closing this for now

**Pull request closed without merging**